sorting/java/HeapSort.java

```diff
@@ -0,0 +1,69 @@
+

```

Comment:
Rename file's name from `HeapSort` to `heap-sort`

Reply:
done